config/extension_import_variables.go

```diff
@@ -61,6 +61,18 @@ func (item *ImportVariables) loadVariablesFromFile(file string) error {
 
 func (item *ImportVariables) loadVariables(newVariables map[string]interface{}, source options.VariableSource) {
 	for _, nested := range item.NestedObjects {
+		for k1, v1 := range newVariables {
```

Comment:
This could be moved outside of the `nested` loop, no? It could be done only once

Reply:
Clearly. Good catch!